docs/en/integrations/language-clients/js.md

```diff
@@ -401,7 +408,60 @@ await client.insert({
 })
 ```
 
-If you have a custom INSERT statement that is difficult to model with this method, consider using [command](#command-method)
+**Example**: Exclude certain columns from the insert statement.
+
+Assuming the table definition such as:
+
+```sql
+CREATE OR REPLACE TABLE mytable
+(id UInt32, message String)
+ENGINE MergeTree()
+ORDER BY (id)
+```
+
+Insert only a specific column:
+
+```ts
+// Generated statement: INSERT INTO mytable (message) FORMAT JSONEachRow
+await client.insert({
+  table: 'mytable',
+  values: [{ message: 'foo' }],
+  format: 'JSONEachRow',
+  // `id` column value for this row will be zero (default for UInt32)
```

Comment:
so one doesn't need to declare `DEFAULT` in DDL to make it work?

Reply:
In some cases, like these primitive types, there is a default value by design (zero, empty string, etc).